src/main/java/com/xpandit/plugins/xrayjenkins/model/ServerConfiguration.java

```diff
@@ -106,7 +96,7 @@ public FormValidation doCheckCredentialId(
         }
             
         if (StringUtils.isBlank(value)) {
-            return FormValidation.error("Authentication not filled!");
+            return FormValidation.warning("Not filling the Credentials is not required, but you will have to fill the in each Build Task.");
```

Comment:
_Filling the Credentials is not required. They will have to be filled in each Build task._